.travis.yml

```diff
@@ -29,6 +30,42 @@ jobs:
+        - secure: jq+9YRcf8bp0WRdIiHGYl8GS0ehQju4Ax4jAIZBPPb8g9D+E9+nCXoqxuG/CyRTnLM5veyz1Xunn39Wm3OVpSaq6ypu913OJ2KbLWAEdgy9XwJ8A1k7h+jPoV3rby7zcNF4dgP/51DSIhzupqGJQYiUEI3Qs8fVSK3DtcrCGALCi2U/zmTxGQYb+dfRbR5Tvzu+BGvGbiz6VpG1ynofONEvO1PpjcDxhQlZNiZ/qM10G0VPh1AqPV2c/wlCfnb2Y97JAluKpOrD0xDY+TdXM1l//mqA6NYO5jaovr16ZX1Xnb/x7lp7Fkec+nE1kYZuY1ylWnKbHU0Hc+tp8KqJ2JXFia3DZGmZ0JszYO7cBBBBsgTvX8UgFSLFrJmCobHnyXQ8WV145QO06Eui3KEVG4S9UZbAQciwmoTFzRRRjEA5Kainhf8Nu7ewSmd2IlSUxDj5T7PmkN5gNK7I/YfhIKZqZTwvY1IQeIswogXwb3Adso1CE0hyiD9siTXTgJN1ew6xI3eN0LL150dip4vmCBP+xt3aJRwgKTmVTlnLoMjFdVN5Gr7ytifRuo9LHt+cuPFWF78dZ0C2j0YTseeQ2JeFLOiJxvWtCMIsgVmPXavZiYOcFIEMF9PqmoLZVE56wznqQoCbHIHl3MMnIKOBN6dc0V7swhB766QRPIYd3bjw=
+        - secure: sDNv5zI4GfrSMODSdRJ+6NbTh7ppg9eLFGlfTS/hneOBF7Zgt9Yi8P71+OehiAqPOGuXe5L6nP3CqgcCFQzl2oUTgl+aU8ZQFSL4nTX5cstc0T06yqMuyzSr9dGbZFYP3Hb27NpJk7KVe/ddOfTcL/MmEsOJqMuzVvklL7GzuxbjlVm6O1Rhc94lsEWctPaS337yOlrc5CKAxFuTl3ohou3Tode9MuZeVd4XKcxrhfZl/8BAYtfV8wim8hwYDXKTBjcOfB7qd8+3Un4tYAXMGjx5wYKOVOoihuim/7sDn0V+YxHKFhXJp8lWa7oITQaGhlMHGNSCNPdQ0nxnntfdKUtD2rxhAjTUXhoaF+bk4mrjRGxgXN80P9rrQjSCN0JyjWx4tnWj2ivjmhGHQZl4swvMp7k9Sikyj0f9KevtsotQBYAQEzTuzPi0kf9uDiZ05/fnttsHpYKw+BpWqbzl5NBQAuJQuoRREwOGOS4CSpybRhS9Bx/i2+lgfT5NXhr76KWD/uN5KurZvuBgbVzdWSXxq4zh6vM2R9iyvgsCN1TzpU/JXfnQDHC8OJMRDqX7pGoYzwEnq6jatRf8ED7yFm86a5hW1u7bWgIgC3TrkCTXcyZvh8lOKl647i9cxCjLfy411PFim8Ic2UY5jMAEMfVwpLV3HcXbZ7ye8HK/bWI=
+        - secure: HGKRJnJ2Y/JURFKWKWNtFpOMk1u99WIest+2SbuDBlDSVEtVWmUjUpAjz+TdaFt1XIaTANiQrHJdmsc4qgQW4QOhqjpOa4pm3CY74OfG7BYsBsaE5Ja8AiCYoiJ57VFejNQQdUMAzV8GWBjpX0CHbSwsG38ObyU6msql+DhLQXtyZZgbVXHq1DwrCAFB3xDazzePzcRuVikIZ70RoM5qa7VLxfWhI1oqHp31il4x3lr+vPYjWDqNulzXXv+8mD9A/GpEtAlDRJ2u2kH/3lSVNkIncUGKbb4Z2OiNUA97V7COLa31lhuDGSTAxVbf9xM9nnEHz/np3z9IHBZMvlCOCIBSPwgClCkUqEp0efKt6OufuAV/TkyFi0Llp4jeaTGeRRfPYF6hAZBbAR+2mOlyom6L6MBw0jrdnJq7euqdS/k/CdxkyMaXGGjqridGRrsRwZ//2ULcXbMgmKehSaqez/1ss4UtnV6LQqqbQQbZQppywz/j9bUoS8XGxaOAfhATRxOLbqQgKYmRD5YnJB+leVwFg3/YpTAiReEMrSwbyEN8OcAenHZtdCg+JwV/ZgobcerBTK0vXcWWlQgttqQEyT41qRU5kvkDfCc1UFDYWkVfaaRs0t+l4W+2YmtzIpDQk081JqFZHz6fXbadAIQu+J8Igap6bp9Y86eMUlLIwc4=
+      install:
+        - curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+      before_script:
+        - az login --service-principal --username "$USER_NAME"  --password "$PASSWORD" --tenant "$TENANT" > /dev/null
+        - chmod +x buildZip.sh
+        - chmod +x version.sh
+        - ./buildZip.sh
+        - mv publish.zip dynatrace-azure-log-forwarder.zip
```

Comment:
it is really needed? if you adding `--path "dynatrace-azure-log-forwarder.zip"` it should also change a name of file